src/driver/mysql/MysqlDriver.ts

```diff
@@ -582,7 +582,7 @@ export class MysqlDriver implements Driver {
     normalizeDefault(columnMetadata: ColumnMetadata): string {
         const defaultValue = columnMetadata.default;
 
-        if ((columnMetadata.type === "enum" || columnMetadata.type === "simple-enum") && defaultValue !== undefined) {
+        if ((columnMetadata.type === "enum" || columnMetadata.type === "simple-enum") && defaultValue != null) {
```

Comment:
`!=` doesn't feel reliable

Reply:
you think `defaultValue !== undefined && defaultValue !== null` is more descriptive? cuz `!=` is reliable, its just not as descriptive since we might have both these values here and each might mean a different think in context.
I will change it if you think that it will help.

Reply:
Perhaps we could just place the null check before all the others? Then `null` is always `NULL` and the rest stays the same. Would apply the same fix for sets too.

```suggestion
        if (defaultValue === null) {
            return `NULL`;
        } else if ((columnMetadata.type === "enum" || columnMetadata.type === "simple-enum") && defaultValue !== undefined) {
```